src/rm.js

```diff
@@ -149,6 +149,11 @@ function _rm(options, files) {
     if (stats.isDirectory() && options.recursive) {
       rmdirSyncRecursive(file, options.force);
     }
+
+    // Path is a symbolic link
+    if (stats.isSymbolicLink()) {
```

Comment:
Ok, that makes sense.

Regarding other file types, here are the types that `fs.lstatSync` recognizes:
```
stats.isFile()
stats.isDirectory()
stats.isBlockDevice()
stats.isCharacterDevice()
stats.isSymbolicLink()
stats.isFIFO()
stats.isSocket()
```
Out of these, I think it only really makes sense to unlink files, directories, symlinks, and maybe fifos. Maybe we should stick to those 4, and ignore anything else?